vnext/templates/templateUtils.js

```diff
@@ -139,6 +140,15 @@ async function updateProjectPackageJson(
   if (options?.logging) {
     console.log(`Modifying ${path.join(projectRoot, 'package.json')}...`);
   }
+
+  // add "windows" to files if not already included
+  if (ensureWindowsInFiles) {
+    props.files = projectPackage.json.files || [];
```

Comment:
We don't want to create a files array if there isn't one to start (it would mess up package publishing for packages that rely on npmignore files instead). So we should check if 'files' exists at all, and only if it does, then make sure that 'windows' gets added to it

Reply:
fixed!